.env.example

```
@@ -5,3 +5,5 @@ NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=
 NEXT_PUBLIC_SUPABASE_ANON_KEY=
 
 BOOKIFIED_BLOB_READ_WRITE_TOKEN=
+
+BOOKIFIED_API_ENDPOINT=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)